attestation-agent/kbs_protocol/src/client/rcar_client.rs

```diff
@@ -135,7 +147,7 @@ impl KbsClient<Box<dyn EvidenceProvider>> {
         let attest_endpoint = format!("{}/{KBS_PREFIX}/attest", self.kbs_host_url);
         let attest = Attestation {
             tee_pubkey,
-            tee_evidence: evidence,
+            tee_evidence: serde_json::from_str(&evidence)?, // TODO: change attesters to return Value?
```

Comment:
see comment above about using serde types directly.

Reply:
is the comment missing from the review?

Reply:
oh indeed. sorry